rotkehlchen/api/v1/schemas.py

```diff
@@ -910,7 +912,7 @@ def make_history_base_entry(
                 data: dict[str, Any],
                 **_kwargs: Any,
         ) -> dict[str, Any]:
-            database = self.context['schema'].database
+            database = CreateHistoryEventSchema.history_event_context.get()['schema'].database
```

Comment:
here too, can't you use self?

```suggestion
            database =self.history_event_context.get()['schema'].database
```